codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/restjson/AWSRestJson1ProtocolGenerator.kt

```diff
@@ -25,9 +26,9 @@ class AWSRestJson1ProtocolGenerator : AWSHttpBindingProtocolGenerator() {
     override val httpResponseGenerator = HttpResponseGenerator(
         unknownServiceErrorSymbol,
         defaultTimestampFormat,
-        AWSRestJson1HttpResponseBindingErrorGeneratable(),
+        HttpResponseBindingOutputGenerator(),
+        AWSRestJson1HttpResponseBindingErrorGeneratable()
```

Comment:
Like the other protocol generators, serdeContext is eliminated and specific renderers are used.